### PR TITLE
feat: change cat channel priorities

### PR DESCRIPTION
**Pull request by cmwaters**
## Description

It seems from talking to Evan that the mempool state channel message should actually be of a higher priority than the tx messages

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use
  [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

**Comment by rootulp** [comment]
I can't approve without more context so defer to @evan-forbes. A GH issue with more context would make this reviewable by others.

**Review by evan-forbes — APPROVED** [review]
I'm not against this change since it make a lot of sense, but I would like to get in the habbit of making very measured decisions for all things gossiping

**Comment by cmwaters** [comment]
Yeah let me put this in draft for now so we can experiment on it

**Review by staheri14 — APPROVED** [review]

**Comment by cmwaters** [comment]
Closing this as no longer relevant

**Pull request closed without merging**
